thorlcr/activities/thactivityutil.cpp

```diff
@@ -783,9 +783,42 @@ IFileIO *createMultipleWrite(CActivityBase *activity, IPartDescriptor &partDesc,
     Owned<IFileIO> fileio;
     if (compress)
     {
+        // rowdif used if recordSize > 0, fallback set to LZ4
+        unsigned __int64 compMethod = COMPRESS_METHOD_LZ4;
+        // fallback override
+        StringBuffer compType;
+        activity->getOpt(THOROPT_FBACK_COMPRESS_TYPE, compType);
```

Comment:
Thanks, I will fix when writing to a published file.
I added fallback type because I left row diff as the default but when the record size is too large or too small the method changes to what was formally LZW and now falls back to LZ4 or whatever you set.  I wanted a way to get back to how is was before without changing the default row diff.  But if we change global default to LZ4 then this can be reworked.